components/I18nFormatters.js

```diff
@@ -9,5 +9,24 @@ export const I18nSupportLink = chunks => (
   <StyledLink href="mailto:support@opencollective.com">{chunks.length || 'support@opencollective.com'}</StyledLink>
 );
 
-const I18nFormatters = { strong: I18nBold, i: I18nItalic, SupportLink: I18nSupportLink };
+export const I18nSignLink = msg => <StyledLink href="/signin">{msg}</StyledLink>;
```

Comment:
Should use `Link` because it's an internal link (so we want frontend routing)

```suggestion
export const I18nSignLink = msg => <Link route="signin">{msg}</Link>;
```

Reply:
done :rocket: